src/components/Form.js

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/booksSlice';
+
+export default function Form() {
+  const dispatch = useDispatch();
+  const [newId, setNewId] = useState('{bookList.length}');
+  const [newTitle, setNewTitle] = useState('');
+  const [newAuthor, setNewAuthor] = useState('');
+
+  const clickAddBook = (e) => {
+    e.preventDefault();
+    dispatch(
+      addBook({
+        id: newId,
+        title: newTitle,
+        author: newAuthor,
+      }),
+    );
+  };
```

Comment:
[Optional] Please consider resetting the form after the submit button is clicked. This will greatly improve the user experience😎